src/content/documentation/develop/debugging.md

```diff
@@ -145,6 +147,8 @@ To get started, open your background script in **Sources**.
 
 As you run your extension, the **Console** displays logged output, including calls to [`console.log()`](https://developer.mozilla.org/docs/Web/API/Console/log) made by the extension's background scripts and any errors the browser raises as it executes these scripts. Note that the console shows all errors raised by the browser, not just errors related to your extension's code.
 
+When you debug a non-persistent background script, the background script won't go idle while the toolbox is open. However, if you need to terminate the background page, you can do so in `about:debugging`.
```

Comment:
IIUC dev tools also has a button.  @rpl

Reply:
@mixedpuppy The "Terminate background script" button is available in the about:debugging addon card and so the way line 150 describes where to find it looks correct (the toolbox is the actual set of tools opened and attached to the extension in a new tab after the extension developer has clicked the "Inspect" button).

We may use `DevTools Toolbox` to refer to the toolbox if we want to better match how the Firefox DevTools team usually refers to it.

Reply:
Thanks @rpl I'll create an issue for the use of `DesTools Toolbox` as it affects the entire article. I assume the "Terminate background script" button only appears for a non-persistent background page - when I have finished the core changes for the developer preview I'll go back and convert the [notify-link-clicks-i18n](https://github.com/mdn/webextensions-examples/tree/master/notify-link-clicks-i18n) example and add a screenshot.